### PR TITLE
claim-god-token-merge-with-execute

### PR DESCRIPTION
**Pull request by shivamgarglab49**

### CODE REVIEW

File: contracts/governance/GovernorCountingSimpleInternal.sol [hunk 1]
```diff
@@ -231,33 +227,17 @@ abstract contract GovernorCountingSimpleInternal is
...
     function _execute(
-        uint256 /* proposalId */,
+        uint256 proposalId,
         address[] memory,
         uint256[] memory,
         bytes[] memory,
         bytes32 /*descriptionHash*/
     ) internal virtual override {
-        //returnGODToken(proposalId);
-        address voter = _msgSender();
```
Comment (amandeeprehal-lab49) [inline contracts/governance/GovernorCountingSimpleInternal.sol:259]: Pls keep this cleanup.

File: deployment/scripts/proposalScript.ts [hunk 2]
```diff
@@ -54,6 +60,10 @@ async function executeProposals(proposals: any[]) {
   console.log("- Proposals execution ended.\n");
 }
 
+async function isProposalActive(proposalId: any) {
+  return Number(await governor.state(proposalId, contractId!)) === 4;
```
Comment (amandeeprehal-lab49) [inline deployment/scripts/proposalScript.ts:64]: Nice work!

File: contracts/governance/GovernorCountingSimpleInternal.sol [hunk 3]
```diff
@@ -284,7 +264,7 @@ abstract contract GovernorCountingSimpleInternal is
         if (responseCode != HederaResponseCodes.SUCCESS) {
             revert("Transfer token failed.");
         }
-        emit GodTokenClaimed(proposalId, address(this), creator);
+        cleanup(creator);
```
Comment (amandeeprehal-lab49) [inline contracts/governance/GovernorCountingSimpleInternal.sol:267]: Why this?

Comment (amandeeprehal-lab49) [reply contracts/governance/GovernorCountingSimpleInternal.sol:267]: cleanup should be inside both _execute and cancelProposal.

Comment (shivamgarglab49) [reply contracts/governance/GovernorCountingSimpleInternal.sol]: changes done.

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**